crates/boot_srv/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "kitsune2_boot_srv"
```

Comment:
`boot` sounds like it's for startup, I'd prefer to spell this one out :)

Reply:
Shall we just go with `bs`? /jk

Reply:
Done: https://github.com/holochain/kitsune2/pull/6/commits/edc254629867487c5bb99ac47f6ebf531dc1d96d